app/models/my_registrations/cnp_module.rb

```diff
@@ -0,0 +1,104 @@
+module MyRegistrations
+  module CnpModule
```

Comment:
@davidlimcheng, can you rename this to be `CancellationNonPaymentModule`?  It will be more helpful to the casual observer.